src/components/PeoplePage.tsx

```diff
@@ -1,35 +1,116 @@
+import React, {
+  memo, useEffect, useMemo, useState,
+} from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { visiblePeople } from '../utils/visiblePeople';
+
+export const PeoplePage: React.FC = memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
```

Comment:
```suggestion
  const [isError, seIsError] = useState(false);
```
or you can keep errorMessage name, but then change catch logic, and set exacly error.message (i mean string type, not boolean)